docs/wiki/Getting started - Setup environment.md

```diff
@@ -322,28 +322,21 @@ To use the pipelines that come with the environment in Azure DevOps, you need to
 
 # 4. Deploy dependencies
 
-At this stage you can execute your first pipeline, that is, the dependency pipeline.
+In order to successfully deploy and test all modules in your desired environment, some modules require resources to be deployed beforehand.
 
-Since the modules we tested often depend on other services, we created a pipeline that provides the modules with various persisting standard services such as virtual networks and key vaults (along with dummy secrets). This _dependency_ pipeline should be prepared and executed before you start running all module pipelines.
-> Note, not all modules require dependencies or only a subset of the deployed.
```

Comment:
Are we sure we should remove that here? It's very important this is updated. @SeSeicht also opened an issue to automate this dependency

Reply:
Just wonder if folks would just execute the pipeline without checking the other page

Reply:
The note is not removed but added to the right section. If folks don't read the how to there's nothing we can do. Otherwise we need to duplicate the content or at least the note. Which is what I understood we wanted to avoid.